pom.xml

```diff
@@ -18,7 +18,7 @@
 
   <groupId>com.adobe.event</groupId>
   <artifactId>xdm-event-model</artifactId>
-  <version>0.92.6-SNAPSHOT</version>
+  <version>0.92.7-SNAPSHOT</version>
```

Comment:
this is to be changed by the release GH action. no need to manually update it

Reply:
@francoisledroff : Incorporated the feedback.
Added a separate profile for java8 builds.